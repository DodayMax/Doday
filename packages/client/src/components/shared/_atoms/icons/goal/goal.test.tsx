import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Goal', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.Goal />);

    expect(icon).toMatchSnapshot();
  });
});
