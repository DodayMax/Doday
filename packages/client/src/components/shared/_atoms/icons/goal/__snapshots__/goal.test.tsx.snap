// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goal renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="black"
        fillRule="evenodd"
        transform="translate(1 1)"
      >
        <path
          d="M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
        />
        <path
          d="M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z"
        />
        <path
          d="M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
        />
        <circle
          cx="11.108"
          cy="10.951"
          r="1"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
          />,
          <path
            d="M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z"
          />,
          <path
            d="M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
          />,
          <circle
            cx="11.108"
            cy="10.951"
            r="1"
          />,
        ],
        "fill": "black",
        "fillRule": "evenodd",
        "transform": "translate(1 1)",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "cx": "11.108",
            "cy": "10.951",
            "r": "1",
          },
          "ref": null,
          "rendered": null,
          "type": "circle",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="black"
          fillRule="evenodd"
          transform="translate(1 1)"
        >
          <path
            d="M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
          />
          <path
            d="M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z"
          />
          <path
            d="M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
          />
          <circle
            cx="11.108"
            cy="10.951"
            r="1"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
            />,
            <path
              d="M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z"
            />,
            <path
              d="M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z"
            />,
            <circle
              cx="11.108"
              cy="10.951"
              r="1"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
          "transform": "translate(1 1)",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M11.108.196C5.168.196.352 5.01.352 10.95s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S17.048.196 11.108.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M11.108 3.618a7.333 7.333 0 1 0 0 14.666 7.333 7.333 0 0 0 0-14.666zm0 13.2a5.867 5.867 0 1 1 0-11.734 5.867 5.867 0 0 1 0 11.734z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M11.108 7.04a3.911 3.911 0 1 0 0 7.822 3.911 3.911 0 0 0 0-7.822zm0 6.356a2.444 2.444 0 1 1 0-4.89 2.444 2.444 0 0 1 0 4.89z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "cx": "11.108",
              "cy": "10.951",
              "r": "1",
            },
            "ref": null,
            "rendered": null,
            "type": "circle",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
